tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -406,9 +406,18 @@ class ExecutionTest {
     TEST_METHOD_PROPERTY(L"Priority", L"2") // Remove this line once warp supports this feature in Shader Model 6.2
   END_TEST_METHOD()
 
-  BEGIN_TEST_METHOD(BarycentricsTest)
-    TEST_METHOD_PROPERTY(L"Priority", L"2")
-  END_TEST_METHOD()
+  TEST_METHOD(BarycentricsTest);
```

Comment:
Since the barycentrics test doesn't properly check for feature support, I think we have to leave this as Priority=2 until it does.